packages/sync/legacy/class.jetpack-sync-users.php

```diff
@@ -9,7 +11,8 @@ class Jetpack_Sync_Users {
 	static $user_roles = array();
 
 	static function init() {
-		if ( Jetpack::is_active() ) {
+		$connection_manager = Jetpack_Sync_Main::init()->get_connection_manager();
```

Comment:
As above, this could just be:

```php
$mgr = new Connection_Manager();
if ( ! $mgr->is_active() ) {
  //...
}
```

Reply:
What did we decide about multiple instances?